exercises/practice/hello-world/contracts/template.fc

```diff
@@ -0,0 +1,11 @@
+#include "imports/stdlib.fc";
+
+() recv_internal(int my_balance, int msg_value, cell in_msg_full, slice in_msg_body) impure {
+  ;; do not code in this block
+}
+
+;; only one get method, which you can call even from your DAPP
+(slice) hello_world() method_id {
+  ;;87521618088882671231069284 is an integer representation of the phrase "Hello world", because strings are hard in Func
+  
+}
```

Comment:
I think this is my only remaining comment. The idea of the `hello-world` exercise is that the student has to make a teeny tiny change (from "Goodbye, Mars!" to "Hello, World!"). The goal is that we check whether the student has the skills to do this fairly simple task, and if not, Exercism isn't the right fit for them. The key part of this is that the change has to be simple. Currently, returning a number does not look very simple to me. Would it be possible to somehow make this easier on the student?

Reply:
Sure! Hello World was written by someone else and I'll change it now!